platform/working-with-vsp/requesting-support.md

```diff
@@ -0,0 +1,17 @@
+# Requesting Support
+
+VSP is here to assist with many issues that VFS teams might face during the development cycle. If you need assistance from someone on VSP, please post your question or request in the #vfs-platform-support Slack channel. 
+
+VSP’s goal is to answer your questions as soon as possible, but if you aren’t getting traction with your request, we have the following escalation path: 
+
+* If you don't get a response within a few hours, @-mention someone from VSP you know in the channel to look at your question. They can then direct the correct person to answer your question if it isn’t them. 
+
+* If you don't get a response within 1 business day, contact your DSVA Product Manager, who will make sure the right people address your question.
+
+We understand the desire to DM someone you’ve worked with previously about a question, but VSP has weekly and biweekly support rotation to limit interruptions for everyone on the platform. 
+
+## Engineering Support
+
+If you are looking specifically for engineering support, you’ll still ask your question in  #vfs-platform-support but the developers that are on the support rotation are also listed in the description of that channel for Back End, Front End and Operations questions. 
```

Comment:
do we want to maybe include a screenshot of this to make it extra-obvious? something like this

![image (1)](https://user-images.githubusercontent.com/769353/78387128-a8271f80-7593-11ea-87f1-f5ac05f3f15c.png)